subsys/shell/Kconfig.template.shell_log_queue_size

```diff
@@ -0,0 +1,10 @@
+config $(module)_LOG_MESSAGE_QUEUE_SIZE
+	int "Log message queue size"
+	default $(default-size)
+	help
+	  Logger threads enqueues log messages which are then processed by shell
```

Comment:
Log queue size... Again, in which units? For example, log strdup buffer size is measured in units of well, units of log_strdup invocation.